### PR TITLE
Enhancements role handbook updates - through enhancements freeze

### PR DESCRIPTION
**Pull request by marosset**
Signed-off-by: Mark Rossetti <marosset@microsoft.com>

#### What type of PR is this:
/kind documentation

#### What this PR does / why we need it:

Part 2 of updates to Enhancements role handbook.
This covers updates up to and including enhancements freeze

#### Which issue(s) this PR fixes:

Part of https://github.com/kubernetes/sig-release/issues/2109

### CODE REVIEW

File: release-team/role-handbooks/enhancements/README.md [hunk 1]
```diff
@@ -131,60 +131,58 @@ It is important that this process be followed and documentation remain up-to-dat
...
 - Add Shadows to the `Enhancements Contact` Field of the Enhancements Tracking Board.
+- Create a Google Sheet to track who will give updates at release team meetings
+  - Access: : **restricted access**, edit rights shared with release team enhancements shadows individually
 
 #### Pre-Enhancements Freeze (Week 1)
 
-- Send an email to the [Kubernetes-Dev](https://groups.google.com/a/kubernetes.io/g/dev) mailing list and a message to #chairs-and-techleads slack channel with a call for enhancements and how to opt-in to the release.
+- Send an email to the [Kubernetes-Dev](https://groups.google.com/a/kubernetes.io/g/dev) mailing list and a message to #chairs-and-techleads slack channel with a call for enhancements and how to opt-in to the release. Previous communications can be used as a template.
+  - [1.26 call for enhancements](https://groups.google.com/a/kubernetes.io/g/dev/c/lnxXgZmOOMo/m/r6QNJIewBwAJ?utm_medium=email)
+  - 1.27 call for enhancements - WIP
```
Comment (leonardpahlke) [inline release-team/role-handbooks/enhancements/README.md:154]:
```suggestion
```

File: release-team/role-handbooks/enhancements/README.md [hunk 2]
```diff
@@ -131,60 +131,58 @@ It is important that this process be followed and documentation remain up-to-dat
     - Until this can be automated; manually create Views with the same Names and Fields from the previous release's project board.
       Tip: You can easily view all the Fields present in each view by clicking the `v` next to the View's name an selecting `Configuration` -> `Fields`
       Note: Remember to **save** each View. If there is a blue bubble next to the Views name there are unsaved changed for that view!
+  - Update automation to add issues to the correct Enhancement Tracking Board
+    - TODO: Work in Progress
```
Comment (leonardpahlke) [inline release-team/role-handbooks/enhancements/README.md:135]: whats missing to describe the sub steps?

File: release-team/role-handbooks/enhancements/README.md [hunk 3]
```diff
@@ -131,60 +131,58 @@ It is important that this process be followed and documentation remain up-to-dat
...
-
-  - For non-participating SIGs, mention the process and deadline for adding Enhancements to the `KEP Collection` tab on the tracking spreadsheet.  An example outreach on Slack could be structured as follows:
-
-    > Hello sig-{{ SIG_NAME }}. Enhancements team here.  Just checking in as we approach enhancements freeze at {{ FREEZE_DATETIME }}. Your SIG has not submitted any enhancements for the {{ CYCLE }} cycle. If your SIG still plans to submit an enhancement, follow the [instructions here](https://groups.google.com/a/kubernetes.io/g/dev/c/slwAhtVqjXk/m/zz-eeXdFBgAJ) so the enhancements team can begin tracking. If your SIG is not planning to submit an enhancement, please reply to this message to confirm. Please reach out to [#release-enhancements](https://kubernetes.slack.com/archives/C02BY55KV7E) if you have any questions.
 
+#### Collection Monitoring and Triage (Weeks 1-4)
+
+- Monitor the Enhancements Tracking Board for new additions as SIGs opt-in KEPs for the release and assign an enhancement contact to each Issue.
```
Comment (leonardpahlke) [inline release-team/role-handbooks/enhancements/README.md:160]: enhancement contact == enhancements shadows and enhancements lead?

File: release-team/role-handbooks/enhancements/README.md [hunk 4]
```diff
@@ -131,60 +131,58 @@ It is important that this process be followed and documentation remain up-to-dat
...
+- For opted in Issues periodically (at least once a week):
+  - Ensure Issue is in the current milestone
+  - Ensure Issue has correct labels (has `tracked/yes`, does not have `tracked/no`, `stage/xxx` is accurate, ...)
+  - Ensure Fields in the Enhancement Tracking Board are populated and accurate (Status, Stage, SIG, ...)
+  - Evaluate if the enhancement satisfies all of the requirements for inclusion in the current release.
```
Comment (leonardpahlke) [inline release-team/role-handbooks/enhancements/README.md:165]: Evaluate.. can we link to a section here?

File: release-team/role-handbooks/enhancements/README.md [hunk 5]
```diff
@@ -131,60 +131,58 @@ It is important that this process be followed and documentation remain up-to-dat
...
+    - Comment on the Issue with a status updating using one of the [templates from below](#enhancement-freeze-templates)
+    - Update `Enhancement Status` field for this Enhancement in the Enhancement Tracking Board (`Tracked` or `At Risk`)
+- Reach out to each [SIG on Slack](https://github.com/kubernetes/community/blob/master/sig-list.md) tagging one or more chairs or technical leads to communicate the Enhancements statuses. Example [communication templates](#sig-outreach-templates) can be found below.
+- Start syncing with Communications Team on giving an induction what's coming up for the release.
 - Send an email to [Kubernetes-Dev](https://groups.google.com/a/kubernetes.io/g/dev) that Enhancement freeze is coming and share current Enhancements status. Examples [1](https://groups.google.com/g/kubernetes-dev/c/-nTNtBBHL2Y/m/WfNzb_E1EAAJ).
+- Provide updates during release team meetings
+  - Use the `KEPs by Stage` insight from the Enhancement Tracking Board ([example](https://github.com/orgs/kubernetes/projects/98/insights/3))
 
-#### Enhancements Freeze (Week 3)
+#### Enhancements Freeze (Week 5)
```
Comment (leonardpahlke) [inline release-team/role-handbooks/enhancements/README.md:174]: the weeks can differ from release to release slightly.. but we can keep it you think its not confusing

File: release-team/role-handbooks/enhancements/README.md [hunk 6]
```diff
@@ -131,60 +131,58 @@ It is important that this process be followed and documentation remain up-to-dat
...
 
 - On Freeze day, send an email to [Kubernetes-Dev](https://groups.google.com/a/kubernetes.io/g/dev) that freeze has happened and upcoming key dates. Examples [1](https://groups.google.com/g/kubernetes-dev/c/JDM7bNKvhqQ/m/8S7BXtXPBQAJ).
-- Remove [SIG Leads Google Group][sig-leads-group]'s access to the `KEP Collection` tab on the enhancement collection sheet.
-- Remove any enhancements that failed to meet the criteria by the Enhancement freeze deadline.
-  - Set their status in the sheet to `Removed from Milestone` and use the `Enhancements` -> `Remove Enhancements from Milestone` menu option to move them over to the `Removed from milestone` tab.
-  - Remove the milestone and change `tracked/yes` label to `tracked/no` on the enhancement issue.
+- Remove any Enhancements that failed to meet the criteria by the Enhancement freeze deadline.
+  - Set their **Enhancement Status** in the board to `Removed from Milestone`.
+  - Remove the milestone and change `tracked/yes` label to `tracked/no` on the Enhancement Issue.
```
Comment (leonardpahlke) [inline release-team/role-handbooks/enhancements/README.md:179]: maybe switch both? :D
```suggestion
  - Remove the milestone and change `tracked/yes` label to `tracked/no` on the Enhancement Issue.
  - Set their **Enhancement Status** in the board to `Removed from Milestone`.
```

Comment (leonardpahlke) [reply release-team/role-handbooks/enhancements/README.md:179]:
```
/remove-label lead-opted-in
/remove-label tracked/yes
/label tracked/no
/milestone clear
```

File: release-team/role-handbooks/enhancements/README.md [hunk 7]
```diff
@@ -131,60 +131,58 @@ It is important that this process be followed and documentation remain up-to-dat
...
 - Clean up Enhancements issues by removing milestone from the enhancements that have not opted-in and make sure that number of in-tree open issues with current milestone matches number of opted-in enhancements.
 - Any enhancements removed from the milestone will now require an exception. As exception requests come in, discuss each with the Release Lead (and Shadows) to arrive at an approve/reject decision.
   - Create an exception file in the Release for exceptions Example [1](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.14/exception.yaml).
-  - If a previously removed Enhancement has had their exception Approved, set their status to `Tracked` and use the `Enhancements` -> `Track Removed Enhancements` menu option to move it back to the `Enhancements` and `Docs` tabs.
+  - If a previously removed Enhancement has had their exception Approved, set their **Enhancement Status** to `Tracked` in the board.
```
Comment (leonardpahlke) [inline release-team/role-handbooks/enhancements/README.md:183]:
```suggestion
  - If a previously removed Enhancement has had their exception Approved, set their **Enhancement Status** to `Tracked` in the enhancements board.
```

Comment (leonardpahlke) [reply release-team/role-handbooks/enhancements/README.md:183]: maybe this extra explicitly is not needed (feel free to drop this comment)

File: release-team/role-handbooks/enhancements/README.md (hunk 2 above)
Comment (marosset) [reply release-team/role-handbooks/enhancements/README.md:135]: We need to get it working first 🤣
AFAIK we are still having issues with the PROW job and hadn't decided if we were going to continue to try to run this in PROW or as a GitHub Action.

File: release-team/role-handbooks/enhancements/README.md (hunk 3 above)
Comment (marosset) [reply release-team/role-handbooks/enhancements/README.md:159]: Correct

File: release-team/role-handbooks/enhancements/README.md (hunk 5 above)
Comment (marosset) [reply release-team/role-handbooks/enhancements/README.md:173]: Should I remove the number here?
If I do that then I can update the above section to `Collection Monitoring and Triage (Weeks 1-Enhancements Freeze)`

File: release-team/role-handbooks/enhancements/README.md (hunk 2 above)
Comment (leonardpahlke) [reply release-team/role-handbooks/enhancements/README.md:135]: 👍 ok

File: release-team/role-handbooks/enhancements/README.md (hunk 5 above)
Comment (leonardpahlke) [reply release-team/role-handbooks/enhancements/README.md:173]: Maybe rephrase headers to : "before Enhancement Freeze", "On Enhancements Freeze Day", "During the week after Enhancements Freeze" and so on... so define time spans. (you can also stick to the current form if it works..)

File: release-team/role-handbooks/enhancements/README.md (hunk 4 above)
Comment (marosset) [reply release-team/role-handbooks/enhancements/README.md:164]: The next line / sub point links to a section with the requirements. Is that sufficient?

File: release-team/role-handbooks/enhancements/README.md (hunk 5 above)
Comment (marosset) [reply release-team/role-handbooks/enhancements/README.md]: Updated - PTAL

File: release-team/role-handbooks/enhancements/README.md (hunk 6 above)
Comment (marosset) [reply release-team/role-handbooks/enhancements/README.md]: updated

File: release-team/role-handbooks/enhancements/README.md (hunk 7 above)
Comment (marosset) [reply release-team/role-handbooks/enhancements/README.md]: updated

File: release-team/role-handbooks/enhancements/README.md (hunk 4 above)
Comment (palnabarun) [reply release-team/role-handbooks/enhancements/README.md:164]: I think it's good to have it explicitly linked here as well.

**Review by palnabarun — COMMENTED** [review]
These are really good improvements to the handbook! Thanks Mark!

Once the conversations are resolved, happy to `lgtm`.

File: release-team/role-handbooks/enhancements/README.md (hunk 4 above)
Comment (leonardpahlke) [reply release-team/role-handbooks/enhancements/README.md:164]: @marosset ^

**Review by leonardpahlke — COMMENTED** [review]
Looks like we have one open discussion otherwise LGTM!

File: release-team/role-handbooks/enhancements/README.md (hunk 4 above)
Comment (marosset) [reply release-team/role-handbooks/enhancements/README.md:164]: OK - let me update this

Comment (marosset) [reply release-team/role-handbooks/enhancements/README.md]: I linked to https://github.com/kubernetes/sig-release/blob/0c8a730269f8031a4a385e1dd827a02a68cffe75/releases/release_phases.md?plain=1#L3-L11 to maintain a single source of truth for this information.

I'll update that section in a separate PR.

PTAL

**Comment by marosset** [comment]
@palnabarun @leonardpahlke - I think I addressed all the feedback - Thansk!

**Review by leonardpahlke — COMMENTED** [review]
/lgtm
Thanks!
